seed/challenges/01-front-end-development-certification/bootstrap.json

```diff
@@ -507,9 +507,9 @@
       "id": "bad87fee1348cd8acef08811",
       "title": "Taste the Bootstrap Button Color Rainbow",
       "description": [
-        "The <code>btn-primary</code> class is the main color you'll use in your app. It is useful for highlighting actions you want your user to take.",
-        "Add Bootstrap's <code>btn-primary</code> class to your button.",
-        "Note that this button will still need the <code>btn</code> and <code>btn-block</code> classes."
+        "The <code>btn-primary</code> class styles app elements you apply it to in order to indicate the primary action on a page. It essentially highlights actions you want your user to take.",
```

Comment:
I think the text without commas is okay. But with commas, I think it would be better as

```diff
- class styles app elements you apply it to in order to indicate
+ class, which styles app elements you apply it on, indicates
```

It's a little less wordy.